zhaquirks/develco/open_close.py

```diff
@@ -16,16 +16,28 @@ class DevelcoPowerConfiguration(PowerConfigurationCluster):
     MAX_VOLTS = 3.0
 
 
-(
-    QuirkBuilder("frient A/S", "WISZB-131")
-    .applies_to("Develco Products A/S", "WISZB-120")
-    .applies_to("frient A/S", "WISZB-120")
-    .applies_to("Develco Products A/S", "WISZB-121")
-    .applies_to("frient A/S", "WISZB-121")
+base_quirk = (
+    QuirkBuilder()
     .replaces(DevelcoIasZone, endpoint_id=35)
     .replaces(DevelcoPowerConfiguration, endpoint_id=35)
     # The binary input cluster is a duplicate
     .prevent_default_entity_creation(endpoint_id=35, cluster_id=BinaryInput.cluster_id)
+)
+
+# Entry Sensor 2 Pro + Entry Sensor (basic), no tamper
+(
+    base_quirk.clone()
+    .applies_to("frient A/S", "WISZB-131")
+    .applies_to("Develco Products A/S", "WISZB-121")
+    .applies_to("frient A/S", "WISZB-121")
```

Comment:
In Zigbee2MQTT, `WISZB-121` (basic Entry Sensor, 1st Gen) supports the tamper sensor, but according to specsheet, it doesn't: https://cdn.prod.website-files.com/67f4edb3e49fc9eedd4d58ff/68627bf514b30153d4133678_Entry%20sensor_frient_v1.0.pdf

Reply:
deCONZ seems to support tamper for the basic sensor (1st Gen) as well: https://github.com/dresden-elektronik/deconz-rest-plugin/blob/41d9a4933f45b6d14fbc96b9df0f8524ffb66c4c/devices/frient/wiszb-121_open_close_sensor.json#L111-L114

I'll adjust this PR to only remove the tamper sensor for the 2nd Gen Pro sensor then.
It would be good to get confirmation on this though 😄

Reply:
Let me check the device itself, I might have one. Every one I added explicitly should have tamper sensor support.